src/cameraGeom/Camera.cc

```diff
@@ -31,6 +31,35 @@ namespace cameraGeom {
 
 namespace {
 
+/**
+ * Get a transform from one TransformMap
+ *
+ * `fromSys` and `toSys` must both be present in the same TransformMap, but that TransformMap may be from
+ *    any detector or this camera object.
+ *
+ * @param[in] fromSys  Camera coordinate system of input points
+ * @param[in] toSys  Camera coordinate system of returned points
+ * @returns an afw::geom::TransformPoint2ToPoint2 that transforms from `fromSys` to `toSys` in the forward
+ *    direction
+ *
+ * @throws lsst::pex::exceptions::InvalidParameter if no transform is available.  This includes the case that
+ *    `fromSys` specifies a known detector and `toSys` specifies any other detector (known or unknown)
+ * @throws KeyError if an unknown detector is specified
```

Comment:
`lsst::pex::exceptions::KeyError`? Docs won't provide a link unless the name has all the qualifiers it would need at this point in the code.